### PR TITLE
Addition of code of Stupid machine in java

### PR DESCRIPTION
**Pull request by raksha009**
Fixes issue #4630 
Code added for STUPMACH problem of codechef platform

@arnavb plz review

### CODE REVIEW

File: code/online_challenges/src/codechef/STUPMACH/Stupmach.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+public class Stupmach {
+
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        int t = s.nextInt();
+        while (t-- >0){
+            int n = s.nextInt();
+            int arr[] = new int[n];
+            for (int i = 0; i<n; i++){
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STUPMACH/Stupmach.java]:
```suggestion
            for (int i = 0; i < n; i++){
```

File: code/online_challenges/src/codechef/STUPMACH/Stupmach.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                arr[i] = s.nextInt();
+            }
+
+            long maxtokens = 0;
+            int min = arr[0];
+            for (int i = 0;i < n; i++){
+                if (min <= arr[i]){
+                    maxtokens += min;
+                }
+                else{
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STUPMACH/Stupmach.java]:
```suggestion
                if (min <= arr[i]) {
                    maxtokens += min;
                }
                else {
```

**Comment by raksha009** [comment]
@arnavb, made the changes plz review.

**Comment by arnavb** [comment]
Nice job!

**Pull request merged**